articles/connections/social/facebook.md

```diff
@@ -1,145 +1,66 @@
 ---
-title: Connect your app to Facebook
+title: Add Facebook Login to Your App
 connection: Facebook
 index: 2
 image: /media/connections/facebook.png
 seo_alias: facebook
-description: This article shows you how to connect your app to Facebook
+description: Learn how to add login functionality to your app with Facebook. You will need to generate keys, copy these into your Auth0 settings, and enable the connection.
 toc: true
 topics:
+  - authentication
   - connections
   - social
   - facebook
 contentType: how-to
 useCase:
-    - customize-connections
-    - add-idp
+  - add-login
+  - customize-connections
+  - add-idp
 ---
-# Connect your app to Facebook
+# Add Facebook Login to Your App
 
-This article describes how to add login with Facebook to your app. It also discusses how you can get an Access Token in order to access the Facebook API.
+This guide will show you how to add functionality to your web app that allows your users to log in with Facebook. Along the way, you will also learn how to get an Access Token that will allow you to access the Facebook API.
 
-First you need to connect your Auth0 application to Facebook. This is summarized in the following steps:
+1. Set up your app in Facebook. To learn how, follow Facebook's [App Development](https://developers.facebook.com/docs/apps) docs. During this process, Facebook will generate an **App ID** and **App Secret** for your application; you can find this in the app's **Basic Settings**.
```

Comment:
Maybe make these numbered steps into sections?

## Set up your app in Facebook

....

## Create a social connection

....

(then test your connection)

## Access the Facebook API

Reply:
https://github.com/auth0/docs/pull/7382 kinda like that